
# HYPOTHESIS TESTING

1-State the Hypotheses statement

Null Hypothesis (H0): The weekly operating cost model is accurate, and the mean weekly cost is equal to the theoretical mean weekly cost according to the cost model.
Alternative Hypothesis (H1): The weekly operating cost model is not accurate, and the mean weekly cost is higher than the theoretical mean weekly cost according to the cost model.

2-Calculate the Test Statistic

Calculate the Theoretical Mean Weekly Cost:

Given the cost model:
    W=1000+5X
    For X=600 units:
Theoretical Mean Cost=1000+5×600=1000+3000=4000

Calculate the Standard Deviation of Weekly Costs:
    σ=5×Standard Deviation of Units Produced
 Given: 
    Standard Deviation of Units Produced=25
    σ=5×25=125
    
Calculate the Test Statistic (t):  
      t = x(bar) −Theoretical Mean Cost/σ/√n
      
      
​Where:
    x=3050 (Sample Mean Weekly Cost)
    Theoretical Mean Cost=4000
    𝜎=125
    σ=125
    n=25 (Sample Size)

    

In [2]:
import numpy as np
from scipy.stats import norm
x_bar = 3050
mu = 1000 + 5 * 600
print("Theoretical Mean Weekly Cost:", mu)
sigma = 5 * 25
print("Standard Deviation:", sigma)
n = 25
t = (x_bar - mu) / (sigma / np.sqrt(n))
print("Test Statistic (t):", t)

Theoretical Mean Weekly Cost: 4000
Standard Deviation: 125
Test Statistic (t): -38.0


3. Determine the Critical Value

Using the significance level α=0.05 for a one-tailed test
    Find the Critical Value:
        Since we are dealing with a one-tailed test, we use the Z-table to find the critical value for α=0.05.
        The critical value for 
        α=0.05 in a one-tailed Z-test is approximately 1.645.

In [4]:
alpha = 0.05
critical_value = norm.ppf(1 - alpha)
print("Critical Value:", critical_value)

Critical Value: 1.6448536269514722


4. Make a Decision

Compare the calculated test statistic with the critical value:
    If t > 1.645: Reject the null hypothesis (H0). There is evidence that the weekly operating costs are higher than predicted by the model
    If t ≤ 1.645: Fail to reject the null hypothesis (H0). There is no significant evidence that the weekly operating costs are higher than predicted.
    

In [5]:
if t > critical_value:
    print("Reject the null hypothesis. There is strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.")
else:
    print("Fail to reject the null hypothesis. There is no strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.")

Fail to reject the null hypothesis. There is no strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.


5. Conclusion

Based on the hypothesis testing, we reject the null hypothesis. There is strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests. The test statistic is 2.1213, and the critical value is 1.6449.

--> Providing  a clear conclusion based on my analysis.

The final answer is that there is strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.